api/metrics/views.py

```diff
@@ -69,10 +70,21 @@ def format_response(self, response, query_params):
             'data': data,
         }
 
-    def execute_search(self, search):
-        # TODO - this is copied from get_count_for_preprint in metrics.py - abstract this out in the future
+    def execute_search(self, search, query=None):
         try:
-            response = search.execute()
+            # There's a bug in the ES python library the prewvents us fro updating the search object, so lets just make
```

Comment:
Typo: `prevents`
Typo: `from`

Reply:
Also, is this bug documented anywhere? If so, could we link to the related ticket?

Reply:
Unfortunately the bug isn't documented anywhere I could find.